docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,264 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_safety:
+
+*****************
+Safety Systems
+*****************
+The following manual is meant to outline the safety systems present on Vertiq modules. Safety systems are built into your module's firmware, 
+and seek to protect your module from things like over-velocity mechanical issues, power issues, and overheating.
+
+.. _protect_against_regen:
+
+Protecting Against Voltage Regeneration Spikes
+====================================================
+What are Regenerative Voltage Spikes
+------------------------------------------------------------
+Regeneration voltage occurs when the commanded module voltage is less than the module's back-EMF voltage. When this occurs, the motor's kinetic 
+energy is converted directly to electrical energy, and produces a current flowing out of the motor and into the power source.
+
+The commanded module voltage may be less than the back-EMF voltage in cases where the module is slowing down or braking. The faster that the commanded voltage 
```

Comment:
It's not a rate relationship, it's a difference relationship.  "A larger spread between back-EMF and commanded module voltage leads to a greater magnitude of regeneration current".